corpus/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .convert import *
 from .costa import *
 from .data_augmentation import *
+from .data_augmentation_audiomentations import *
```

Comment:
would it be sensible to include this in `corpus/data_augmentation` instead of creating a new module?

Reply:
I created a new module because this functionality depends on a different external Python library (audiomentations), so that it doesn't break the code of the old module for people who don't have this library installed.

Reply:
I would actually to prefer to have it in `data_augmentation`, you can use try/except statements to check for missing imports and just give a warning. You can check at the top of adaptation/ivector.py for an example. But this is not a necessity, just my personal opinion.

Reply:
Now after merging the files, this line should be removed.